### ISSUE TITLE
Seed CSR missing from CSR listings

### ISSUE THREAD

**Issue by Timmmm** [riscv/riscv-isa-manual#1747]
The CSR listings in the privileged spec are missing [`seed` (0x015)](https://riscv-specs.timhutt.co.uk/spec/riscv-isa-release-f3f8a93-2024-11-28/riscv-unprivileged.html#crypto_scalar_seed_csr).

**Comment by aswaterman** [comment]
Indeed.  I'd happily review a PR that rectifies this omission, else I'll get around to it eventually.
